packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1100,7 +1093,6 @@ export abstract class Chart extends Observable implements AgChartInstance {
 
     protected onLeave(event: InteractionEvent<'leave'>): void {
         if (!this.tooltip.pointerLeftOntoTooltip(event)) {
-            this.disablePointer();
```

Comment:
NOTE: On mouse leaving the canvas bounding-box, the highlighted selection + tooltip should be cleared.